src/CoreShop/Component/Index/Filter/RangeFilterConditionProcessor.php

```diff
@@ -37,8 +37,8 @@ public function prepareValuesForRendering(FilterConditionInterface $condition, F
             'label' => $condition->getLabel(),
             'minValue' => $minValue,
             'maxValue' => $maxValue,
-            'currentValueMin' => $currentFilter[$field . '-min'] ? $currentFilter[$field . '-min'] : $minValue,
-            'currentValueMax' => $currentFilter[$field . '-max'] ? $currentFilter[$field . '-max'] : $maxValue,
+            'currentValueMin' => ('' !== (string)$currentFilter[$field . '-min']) ? $currentFilter[$field . '-min'] : $minValue,
```

Comment:
I think we should check if the index `$currentFilter[$field . '-min']` actually is set here? PHP with strict-types would trigger a warning here

Reply:
Resolved in a new commit.